sdk/security/keyvault/azcertificates/example_test.go

```diff
@@ -94,3 +96,59 @@ func ExampleClient_DeleteCertificate() {
 	// In a soft-delete enabled vault, deleted resources can be recovered until they're purged (permanently deleted).
 	fmt.Printf("Certificate will be purged at %v", *resp.ScheduledPurgeDate)
 }
+
+// This example uses `ImportCertificate` to import an existing PFX certificate.
+func ExampleClient_ImportCertificate_pfx() {
+	// Getting certificate data from locally stored PFX file then encoding to base64
+	data, err := os.ReadFile("PFX_CERT_PATH")
+	if err != nil {
+		// TODO: handle error
+	}
+	encodedData := base64.StdEncoding.EncodeToString(data)
+
+	// Assuming you already have a PFX containing your key pair, you can import it into Key Vault.
+	// You can do this without setting a policy, but the policy is needed if you want the private key to be exportable
+	// or to configure actions when a certificate is close to expiration.
+	parameters := azcertificates.ImportCertificateParameters{
+		Base64EncodedCertificate: to.Ptr(encodedData),
+		CertificatePolicy: &azcertificates.CertificatePolicy{
+			SecretProperties: &azcertificates.SecretProperties{
+				ContentType: to.Ptr("application/x-pkcs12"),
+			},
+		},
+	}
+
+	resp, err := client.ImportCertificate(context.TODO(), "pfxCertName", parameters, nil)
+	if err != nil {
+		// TODO: handle error
+	}
+
+	fmt.Printf("PFX certificate %s imported successfully.", resp.ID.Name())
+}
+
+// This example uses `ImportCertificate` to import an existing PEM certificate.
+func ExampleClient_ImportCertificate_pem() {
+	// Getting certificate data from locally stored PEM file. Contents of .pem file are already base64 encoded
+	data, err := os.ReadFile("PEM_CERT_PATH")
+	if err != nil {
+		// TODO: handle error
+	}
+
+	// To import a PEM-formatted certificate, you must provide a CertificatePolicy that sets the ContentType to
+	// CertificateContentType.pem or the certificate will fail to import (the default content type is PFX).
```

Comment:
Do we have a `CertificateContentType` enum? If we do, this example should show it. In any case I'd delete this comment because I think the example clearly implies Key Vault requires a specific value for `ContentType`

Reply:
No CertificateContentType, you have to know the string